trajectory_executor/src/trajectory_executor/trajectory_executor.cpp

```diff
@@ -43,10 +43,12 @@ namespace trajectory_executor
 
         out[default_control_plugin_] = default_control_plugin_topic_;
 
-        //Hardcoding jerk control wrapper
-        std::string default_jerk_control_plugin = "Pure Pursuit Jerk";
-        std::string default_jerk_control_plugin_topic_ = "/guidance/pure_pursuit/plan_jerk_trajectory";
-        out[default_jerk_control_plugin]=default_jerk_control_plugin_topic_;
+        //Hardcoding platoon control plugins
+
+        std::string control_plugin2 = "PlatooningControlPlugin";
```

Comment:
Didn't we update this to be dynamic? If these are just defaults, let's update that comment and then return this to pure_pursuit.

Reply:
No I didnt have time to make it dynamic, so I just added the platooning to the default one that was already set. I think it is better to have a separate story to handle the dynamic update.